python/lsst/ts/hvac/xml/hvac_mqtt_to_SAL_XML.py

```diff
@@ -399,7 +399,8 @@ def _generic_collect_topics_and_items(row, topics, items):
                     elif row["unit"] == "%":
                         unit = "%"
                     else:
-                        raise ValueError(f"Unknown unit {row['unit']}")
+                        # raise ValueError(f"Unknown unit {row['unit']}")
+                        unit = ""
```

Comment:
Why this change? Can you at least log a warning if this occurs? Silently ignoring errors is not safe.

Reply:
See my previous comment. Change reverted.